README.md

```diff
@@ -53,8 +53,8 @@ If you want to see the logs replace it with `--sig-proxy=false`.
 * `--volume` -> Binds the satisfactory config folder to the folder you specified.
 Allows you to easily access your savegames.
 * For the environment (`--env`) variables please see [here](https://github.com/wolveix/satisfactory-server#environment-variables).
-* `--memory-reservation` -> Is a memory soft limit.
-* `--memory 16G` -> Limits the RAM that the container uses to 16 Gigabytes.
+* `--memory-reservation=4G` -> Is a memory soft limit.
+* `--memory 6G` -> Limits the RAM that the container uses to 16 Gigabytes.
```

Comment:
```suggestion
* `--memory 6G` -> Limits the RAM that the container uses to 6 Gigabytes.
```

Reply:
Just a nit pick.  Once that's added lgtm

Reply:
Thanks 😀